src/Form/selectors-css.js

```diff
@@ -24,13 +26,20 @@ input:not([type])[aria-label*=email i]:not([aria-label*=search i]),
 input[name=username][type=email],
 input[autocomplete=username][type=email],
 input[autocomplete=username][placeholder*=email i],
-input[autocomplete=email]`
+input[autocomplete=email]`,
+    // https://account.nicovideo.jp/login
+    `input[name="mail_tel" i]`
+]
 
 // We've seen non-standard types like 'user'. This selector should get them, too
 const GENERIC_TEXT_FIELD = `
 input:not([type=button]):not([type=checkbox]):not([type=color]):not([type=date]):not([type=datetime-local]):not([type=datetime]):not([type=file]):not([type=hidden]):not([type=month]):not([type=number]):not([type=radio]):not([type=range]):not([type=reset]):not([type=search]):not([type=submit]):not([type=time]):not([type=url]):not([type=week])`
 
-const password = `input[type=password]:not([autocomplete*=cc]):not([autocomplete=one-time-code]):not([name*=answer i]):not([name*=mfa i]):not([name*=tin i])`
+const password = [
+    `input[type=password]:not([autocomplete*=cc]):not([autocomplete=one-time-code]):not([name*=answer i]):not([name*=mfa i]):not([name*=tin i])`,
+    // DDG's CloudSave feature https://emanuele.duckduckgo.com/settings
+    'input.js-cloudsave-phrase'
```

Comment:
Do we need this specific selector once we've added `autocomplete="new-password"`?